app/src/modules/settings/routes/data-model/field-detail/field-detail-advanced/field-detail-advanced-validation.vue

```diff
@@ -35,6 +18,44 @@ const { t } = useI18n();
 
 const validation = syncFieldDetailStoreProperty('field.meta.validation');
 const validationMessage = syncFieldDetailStoreProperty('field.meta.validation_message');
+
+const validationSync = computed({
+	get() {
+		return {
+			...(validation.value && { validation: validation.value }),
+			...(validationMessage.value && { validationMessage: validationMessage.value }),
```

Comment:
These statements will fail if the left hand of the expression is `falsy` if this happens it will try to spread the falsy value and error with a syntaxError for example `{ ...undefined }`
```suggestion
			...(validation.value ? { validation: validation.value } : {}),
			...(validationMessage.value ?? { validationMessage: validationMessage.value } : {}),
```

Reply:
Apparently that was some outdated knowledge on my part and this seems to fail gracefully now
![image](https://github.com/directus/directus/assets/9389634/f45a4c02-065d-4864-89f6-dda3c5cb3579)

Reply:
Thank you for checking ❤️